src/libs/DateUtils.ts

```diff
@@ -668,6 +669,22 @@ function formatWithUTCTimeZone(datetime: string, dateFormat: string = CONST.DATE
     return '';
 }
 
+/**
+ *
+ * @param timezone
+ * function parse timezone to supported timezone
+ * @returns Timezone
+ */
+function parseTimezone(timezoneInput: Timezone): Timezone {
```

Comment:
I think we should name it `formatToSupportedTimezone` or `convertToSupportedTimezone`?